templates/lvl-max/app/sim_test.go.tmpl

```diff
@@ -0,0 +1,296 @@
+package app
+
```

Comment:
not sure if we need this file as it has links to the cosmos-sdk in particular

Reply:
I see.
I often want this file in the flow like below:

- scaffolding chain
- scaffolding original module
- chain uses the module
- testing the chain

copy-and-paste-ing integration test of module everytime is bothersome.

It is my opinion but I'll follow your final dicision.

Reply:
okay we can keep it but `simapp.AppStateFn(app.Codec(), app.SimulationManager())` is using the default genesis of simapp i believe so if you add new modules you will have to add in these files manually. We can merge it as is now, and I will do a longer simulation follow up to app-lvl1 and this one

Reply:
Okay, thank you very much.